keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueWorkerClient.java

```diff
@@ -95,4 +96,39 @@ public ByteBuffer call() throws TachyonTException, TException {
       }
     });
   }
+
+  /**
+   * Gets a batch of keys next to the current key in the partition.
+   * If current key is null, it means get the initial batch of keys.
+   * If there are no more next keys, an empty list is returned.
+   *
+   * @param blockId the id of the partition
+   * @param currentKey the current key
```

Comment:
maybe rename this to just "key" since it's only the current key if we're iterating, and this method doesn't need to know that it's being used for iteration.